### PR TITLE
BUGFIX: BB-1265 JP, FR: Previous period and period ago is translated the same

### PR DESCRIPTION
**Pull request by sitina**
* Fix of Japanese translation
* The space in the translation is intentional upon the request of the translator

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2053/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1515/ : FAILURE in 2m 09s

**Comment by sitina** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2055/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1517/ : FAILURE in 1m 51s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2056/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1518/ : SUCCESS in 2m 01s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2062/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1521/ : SUCCESS in 1m 43s

File: src/translations/ja-JP.json [hunk 1]
```diff
@@ -49,5 +49,5 @@
    "gs.list.noItemsFound": "アイテムが見つかりません。",
    "gs.list.error": "リストの読み込みエラー",
    "measure.title.suffix.same_period_year_ago": "前年同期",
-   "measure.title.suffix.previous_period": "前期"
-}
\ No newline at end of file
+   "measure.title.suffix.previous_period": " 期前"
```
Comment (sitina) [inline src/translations/ja-JP.json]: the space here is intentional, the translator has requested that

**Comment by gdgate** [comment]
Build failed (check-teardown pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-stop-zuul-docker/235/ : FAILURE in 4m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2084/ : SUCCESS in 48s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1542/ : SUCCESS in 3m 52s

**Comment by petrjanu** [comment]
Sorry for the mess...I accidentaly force pushed to your branch and deleted your original commits. Created new ones in my fork - new PR is here https://github.com/gooddata/gooddata-react-components/pull/660

**Pull request closed without merging**